docs/index.md

```diff
@@ -32,10 +32,10 @@ This is a Web API reference application designed to "fork and code" with the fol
 
 ### [dotnet Implementation](DotNet-Core-Developer.md)
 
-### [Java Implementation](Java-SpringBoot-Developer.md)
+### [Java Implementation](Java-Spring-WebFlux-Developer.md)
 
 ## CI-CD Documentation
 
-### [ACR Tasks](CI-CD/ACRTask.md)
+### [ACR Tasks](CI-CD/ACR.md)
 
-### [GitHub Actions](CI-CD/GithubActions.md)
+### [GitHub Actions](CI-CD/Docker.md)
```

Comment:
why is the file named docker.md instead of github.md or githubactions.md?  We're using github for ci-cd, so the name is confusing to me.

Reply:
Renamed the file to GitHubActions.md and updated the reference to the file in index.md to prevent link checker from failing.